vcd/resource_vcd_nsxt_network_dhcp_test.go

```diff
@@ -111,6 +117,8 @@ resource "vcd_nsxt_network_dhcp" "pools" {
     start_address = "7.1.1.100"
     end_address   = "7.1.1.110"
   }
+
+  dns_servers = ["8.8.8.8", "8.8.4.4"]
```

Comment:
Will the test work if I run it on version lower than 10.3.1?

Reply:
You are right. The test it's going to fail for versions lower than 10.3.1. Should I create the `dns_servers` var conditionally depending version?

Reply:
Two options:
* Either another test with 10.3.1 specific features
* Either you can add a conditional "TestStep" (using `TestStep.SkipFunc`). Just have in mind that calling `templateFill` in the setup phase also creates a binary test definition so it also needs to be skipped for older versions

Reply:
@Didainius I've created another test step that has set `TestStep.SkipFunc`, so if version is lower than 10.3.1 that testStep is not executed.
Also I conditionally create the config text using templateFill if vcd version is equal or greater than 10.3.1